### PR TITLE
feat: add onlyOnce auth param

### PR DESCRIPTION
**Pull request by tsaleksandrova**

### CODE REVIEW

**Comment by maximnaidenov** [comment]
Seems I missed to explain the main idea - we need to have a generic parameter on config root level, like authOnce: true that fixes the current behaviour. But there is no meaning to have it per-auth config, this is pretty generic.

**Comment by tsaleksandrova** [comment]
so the change would be to rename the key from config.auth.onlyOnce to config.authOnce?

File: src/authenticator/basicUrlAuthenticator.js [hunk 1]
```diff
@@ -15,6 +16,10 @@ function BasicUrlAuthenticator(config,instanceConfig,logger){
 
   this.user = instanceConfig.user;
   this.pass = instanceConfig.pass;
+  
+  if (config.authOnce) {
```
Comment (maximnaidenov) [inline src/authenticator/basicUrlAuthenticator.js:20]: why do you need to implement it in every authenticator, why not only centrally?

File: e2e/scenario/formauth.spec.js [hunk 2]
```diff
@@ -60,4 +60,17 @@ describe('FormAuth scenario test', function() {
             confjs: './scenario/formauth_authorize.conf.js'
         });
     },80000);
+
+    it('should execute auth only for the first test', () => {
+        // will execute 2 spec files:
+        // authOnce will change the page by a click;
+        // authOnce2 will check that the change is still there - which would mean that there wasn't a second auth
+        return Runner.execTest({
+            specs: './scenario/fixture/authOnce*.spec.js',
```
Comment (maximnaidenov) [inline e2e/scenario/formauth.spec.js:69]: please set them explicitly as array of strings

File: e2e/scenario/fixture/authOnce2.spec.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+describe('authOnce2', function() {
```
Comment (maximnaidenov) [inline e2e/scenario/fixture/authOnce2.spec.js:1]: could we name the files as authOnceFirstSpec and  authOnceSecondSpec

File: e2e/scenario/fixture/apps/formauth/authOnce.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE HTML>
```
Comment (maximnaidenov) [inline e2e/scenario/fixture/apps/formauth/authOnce.html:1]: this file looks as subset of app.html, can't we use app.html directly ?.
If necessary you can extend it with the footer, this won't break existing tests

File: src/authenticator/basicUrlAuthenticator.js (hunk 1 above)
Comment (venkatui5) [reply src/authenticator/basicUrlAuthenticator.js]: Can you please share UIveri5 developer document venkatkccs53@gmail.com

**Pull request merged**